ui-admin/src/study/surveys/SurveyEditorView.tsx

```diff
@@ -101,6 +103,28 @@ const SurveyEditorView = (props: SurveyEditorViewProps) => {
           </h5>
         </div>
         { savingDraft && <span className="detail me-2 ms-2">Saving draft...</span> }
+        { validationErrors && !isEmpty(validationErrors) &&
```

Comment:
based on my read of the docs, isEmpty returns true for null/undefined
```suggestion
        { !isEmpty(validationErrors) &&
```